Index: insert.css
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>body {\r\n  font-family: sans-serif;\r\n  margin: 0;\r\n  padding: 0;\r\n  overflow-x: hidden;\r\n}\r\n.container {\r\n  display: flex;\r\n  width: auto;\r\n  max-width: 100%;\r\n  margin: 0 auto;\r\n}\r\n.playback {\r\n  border: 1px solid black;\r\n  width:20%;\r\n  position: relative;\r\n  min-height: 100vh;\r\n  }\r\n.header{\r\n  border: 1px solid black;\r\n  margin-top:0;\r\n  height:70px;\r\n  position: relative;\r\n  min-width: 100%;\r\n  padding-left: 15px;\r\n  padding-top: 8px;\r\n}\r\n.explore{\r\n  border: 1px solid black;\r\n  width:60%;\r\n  position: relative;\r\n  min-height: 100vh;\r\n}\r\n.menu{\r\n  border: 1px solid black;\r\n  position: relative;\r\n  min-height: 100vh;\r\n  min-width:20%;\r\n}\r\n\r\n\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/insert.css b/insert.css
--- a/insert.css	
+++ b/insert.css	
@@ -1,8 +1,9 @@
 body {
   font-family: sans-serif;
-  margin: 0;
-  padding: 0;
+  margin:0;
+  padding:0;
   overflow-x: hidden;
+  shape-image-threshold: inherit;/*If images are not aligning try removing this line */
 }
 .container {
   display: flex;
@@ -16,6 +17,12 @@
   position: relative;
   min-height: 100vh;
   }
+  .record{
+    padding-left:50%;
+    height:89%;
+    background-color:grey;
+    border-radius:50%;
+  }
 .header{
   border: 1px solid black;
   margin-top:0;
